src/Walker/Collector.C

```diff
@@ -113,33 +59,46 @@ Collector::chareOrdPDF( const std::vector< tk::UniPDF >& updf,
   for (const auto& p : tpdf) m_ordtpdf[i++].addPDF( p );
 
   // If all chares on my PE have contributed, send partial sums to host
-  if (m_nopdf == m_nchare) {
+  if (m_nord == m_nchare) {
+
+    // Create Charm++ callback function for reduction
+    CkCallback c1( CkReductionTarget( Distributor, estimateOrd ), m_hostproxy );
```

Comment:
They are both call-backs used immediately in their following lines and never again. Plus every line is commented. I refuse to believe that you do not understand their meaning.